src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+const listUl = document.querySelector('ul');
+
+function convertSalary(salary) {
+  const noCurrency = salary.slice(1);
+
+  return +noCurrency.split(',').join('');
+}
+
+function sortList(salary) {
+  const sort = salary.sort((a, b) =>
+    convertSalary(b.dataset.salary) - convertSalary(a.dataset.salary));
+
+  return sort;
+}
```

Comment:
try to use more meaningful namings and there is no need to create additional variables
```suggestion
function sortList(list) {
  return [...list].sort((a, b) =>
    convertSalary(b.dataset.salary) - convertSalary(a.dataset.salary));
}
```